docs/book/v4/getting-started/features.md

```diff
@@ -30,45 +30,32 @@ With Mezzio, you can build middleware applications such as the following:
 
 ## Features
 
-Mezzio builds on [laminas-stratigility](https://docs.laminas.dev/laminas-stratigility/)
-to provide a robust convenience layer on which to build applications. The
-features it provides include:
+Mezzio builds on [laminas-stratigility](https://docs.laminas.dev/laminas-stratigility/) (Stratigility) to provide a robust convenience layer on which to build applications.
+The features it provides include:
 
-- **Routing**
+### Powerful Routing
 
-  Stratigility provides limited, literal matching only via its
-  `PathMiddlewareDecorator`. Mezzio allows you to utilize dynamic routing
-  capabilities from a variety of routers, providing much more fine-grained
-  matching capabilities. The routing layer also allows restricting matched
-  routes to specific HTTP methods, and will return "405 Not Allowed" responses
-  with an "Allow" HTTP header containing allowed HTTP methods for invalid
-  requests.
+Stratigility provides limited, literal route matching via its `PathMiddlewareDecorator`.
+Mezzio, however, allows you to utilize dynamic routing capabilities from a variety of router packages, providing much more fine-grained matching capabilities.
+Its routing layer also allows restricting matched routes to specific HTTP methods, and will return a [405 Not Allowed][405-not-allowed-url] response with an [Allow HTTP header][allow-http-header-url] containing the allowed HTTP methods for invalid requests.
 
-  Routing is abstracted in Mezzio, allowing the developer to choose the
-  routing library that best fits the project needs. By default, we provide
-  wrappers for Aura.Router, FastRoute, and the laminas-router.
+Routing is abstracted in Mezzio, allowing the developer to choose the routing library that best fits the project needs.
+By default, we provide wrappers for [Aura.Router][aura-router-url], [FastRoute][fastroute-url], and the [laminas-router][laminas-router-url].
```

Comment:
Aura Router is dead: https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2024-08-12-TSC-Minutes.md#abandon-mezziomezzio-aurarouter